data/footnotes.csv

```diff
@@ -827,7 +827,7 @@ Les ministères et organismes doivent indiquer en quoi les activités de program
 * Secrétariat du Conseil du Trésor
 * École de la fonction publique du Canada"
-241,"gov","gov","2018-19",,"PLANNED_EXP","Planned expenditures for most organizations are sourced from Departmental Plans. Where [Planned Spending]{{gl ""PLANNED_EXP""}} information is not available (notably for [Parliamentary Entities]{{gl ""PARL_ORG""}}, the Office of the Governor General's Secretary, the Canadian Security Intelligence Service, the Communication Security Establishment, the International Joint Commission (Canadian Section) and most [Crown Corporations]{{gl ""CROWN_SA""}}, **for illustrative purposes only**, the InfoBase presents the most recent Main Estimates figures for these organizations as being constant across the three planning years.","Pour la plupart des organisations, les données sur les [dépenses prévues]{{gl ""PLANNED_EXP""}} proviennent des plans ministériels. Dans les cas où ces renseignements ne sont pas disponibles (notamment pour les [entités parlementaires]{{gl ""PARL_ORG""}}, le Bureau du Secrétariat du gouverneur général, le Service canadian du renseignement de sécurité, le Centre de la sécurité des télécommunications et la Commission mixte internationale (section canadienne),  ainsi que la plupart des [sociétés d'État]{{gl ""CROWN_SA""}}, l'InfoBase suppose que les dépenses prévues du plus récent Budget principal des dépenses demeureront constantes pour les trois années de planification prévues pour ces organisations. **Ceci est fait à titre indicatif seulement.**"
+241,"gov","gov","2018-19",,"PLANNED_EXP","Planned expenditures for most organizations are sourced from Departmental Plans. Where [Planned Spending]{{gl ""PLANNED_EXP""}} information is not available (notably for [Parliamentary Entities]{{gl ""PARL_ORG""}}, the Office of the Governor General's Secretary, the Canadian Security Intelligence Service, the Communication Security Establishment, the International Joint Commission (Canadian Section) and most [Crown Corporations]{{gl ""CROWN_SA""}} ), **for illustrative purposes only**, the InfoBase presents the most recent Main Estimates figures for these organizations as being constant across the three planning years.","Pour la plupart des organisations, les données sur les [dépenses prévues]{{gl ""PLANNED_EXP""}} proviennent des plans ministériels. Dans les cas où ces renseignements ne sont pas disponibles (notamment pour les [entités parlementaires]{{gl ""PARL_ORG""}}, le Bureau du Secrétariat du gouverneur général, le Service canadian du renseignement de sécurité, le Centre de la sécurité des télécommunications et la Commission mixte internationale (section canadienne),  ainsi que la plupart des [sociétés d'État]{{gl ""CROWN_SA""}} ), l'InfoBase suppose que les dépenses prévues du plus récent Budget principal des dépenses demeureront constantes pour les trois années de planification prévues pour ces organisations. **Ceci est fait à titre indicatif seulement.**"
```

Comment:
`ons]{{gl ""CROWN_SA""}} ), **for`
I don't think we need space before the closing brackets?